include/assimp/Vertex.h

```diff
@@ -183,29 +183,6 @@ struct Vertex {
         return *this;
     }
 
-    bool operator < (const Vertex & o) const {
```

Comment:
I moved the updated comparison logic to `JoinVerticesProcess.cpp` since it is specific to that, and I wanted to avoid causing any confusion or surprises. Pre 12f33099964c50877ff9bc29957212e020deb6be assimp did not define `Vertex::operator<`.